### DISCUSSION TITLE
Array Like Data in Tiddlywiki

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6357] [Q&A] [answered]
There is a discussion on Talk: https://talk.tiddlywiki.org/t/request-simple-array-like-data-in-tiddlywiki/1861
requesting for array data in Tiddlywiki!

While dictionary-data tiddler are kind of array/dictionary data, is it feasible to have core support for array type data in Tiddlywiki?

**Comment by Jermolene** [comment]
The TiddlyWiki name for an array is a list. We have a fairly complete set of tools for manipulating lists, is there anything in particular that's hard to achieve?

**Reply by kookma** [reply]
> The TiddlyWiki name for an array is a list. We have a fairly complete set of tools for manipulating lists, is there anything in particular that's hard to achieve?

Thank you Jeremy! While I was looking for a more direct wat to manipulate arrays! but with the [solutions ](https://talk.tiddlywiki.org/t/request-simple-array-like-data-in-tiddlywiki/1861/4)given on Talk, I think we can manage most cases! macros and variables let us to have dynamic data definition of arrays!

**Comment by Jermolene** [comment, marked as answer]
(I also answered at the TalkTW thread)
